src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:not(:last-child) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
You can transfer properties for all modifiers because now you have only for 4-star case.